Buildings/Components/Walls/Pillars.gd

```diff
@@ -0,0 +1,56 @@
+extends Node3D
+
+
+# Needs to be set prior to build
+var ground_height_at_center
+var height
+var floors: int :
+	set(new_floors):
+		floors = new_floors
+		height = floor_height * new_floors
+var floor_height = 2.5
+
+
+func build(footprint: Array):
+	var mm: MultiMesh = MultiMesh.new()
+	mm.transform_format = MultiMesh.TRANSFORM_3D
+	mm.set_instance_count(footprint.size() * floors)
```

Comment:
I think the `* floors` needs to be removed, since there is only one layer of pillars which are scaled vertically